mlflow/pyfunc/__init__.py

```diff
@@ -534,6 +534,8 @@ def save_model(path, loader_module=None, data_path=None, code_path=None, conda_e
 
                       If ``None``, no artifacts are added to the model.
     """
+    if 'mlflow_model' in kwargs:
```

Comment:
We should also validate that `model` is the only keyword argument specified. Otherwise, users who pass incorrect keyword arguments (typos, etc) will encounter silent failures. Maybe we can do something like:

```
model = kwargs.pop('mlflow_model', model)
if len(kwargs) > 0:
    raise TypeError("save_model() got unexpected keyword arguments: {}".format(kwargs))
```